<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraManipUI</class>
 <widget class="QFrame" name="CameraManipUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="m_createCamera">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Create Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_useCamera">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Use Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_saveCamera">
         <property name="text">
          <string>Save Camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_addKeyFrame">
         <property name="text">
          <string>Add Key Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_showFrames">
         <property name="text">
          <string>Show Frames</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_playPath">
         <property name="text">
          <string>Play Path</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_resetPath">
         <property name="text">
          <string>Reset Path</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="m_savePath">
         <property name="text">
          <string>Save Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loadPath">
         <property name="text">
          <string>Load Path</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
